File: Examen de Lenguaje/C/Cronometro.c
#include <stdio.h>
#include <windows.h>  // Para Sleep()

int main() {
    int horas = 0, minutos = 0, segundos = 0;

    printf("Cronometro iniciado (Ctrl+C para detener)\n");
    while (1) {
        Sleep(1000);  // espera 1000 ms = 1 segundo
        segundos++;

        if (segundos == 60) {
            segundos = 0;
            minutos++;
        }

        if (minutos == 60) {
            minutos = 0;
            horas++;
        }

        printf("\r%02d:%02d:%02d", horas, minutos, segundos);
        fflush(stdout);
    }

    printf("\nPresione ENTER para salir...");
    getchar();
    return 0;
}

File: Examen de Lenguaje/Fortran/Cronometro.f90
program Cronometro
    implicit none
    integer :: horas, minutos, segundos
    character(len=1) :: pausa

    horas = 0
    minutos = 0
    segundos = 0

    print *, "Cronometro iniciado (Ctrl+C para detener)"
    do
        call sleep(1)  ! Espera 1 segundo
        segundos = segundos + 1

        if (segundos == 60) then
            segundos = 0
            minutos = minutos + 1
        end if

        if (minutos == 60) then
            minutos = 0
            horas = horas + 1
        end if

        write(*,'(I2.2, ":", I2.2, ":", I2.2)') horas, minutos, segundos
    end do

    print *, "Presione ENTER para salir..."
    read(*,*) pausa
end program Cronometro

